common/vts.proto

```diff
@@ -15,6 +19,122 @@ message CloseArgs {
 }
 
 message CloseResponse {
+    Status status = 1;
+}
+
+message OpenArgs {
+}
+
+message OpenResponse {
+    Status status = 1;
+}
+
+enum TaType {
+    TA_UNSPECIFIED = 0;
+    TA_RAWPUBLICKEY = 1;
+    // Refer https://github.com/veraison/veraison/issues/76, to introduce certs here!
+    TA_INVALID = 255;
+}
+
+message EndorsementID {
+    AttestationFormat type = 1;
+    google.protobuf.Struct parts = 2;
+}
+
+message Evidence {
+    google.protobuf.Struct value = 1;
+}
+
+message GetEndorsementsRequest{
+    EndorsementID id = 1;
+    Evidence evidence = 2;
+}
+
+message Status {
+    bool result = 1;
+    string error_detail = 2;
+}
+
+message GetEndorsementsResponse {
+    Status status = 1;
+    string details = 2;
+    TrustVector trust_vector = 3;
+    google.protobuf.Struct endorsements = 4;
+}
+
+
+message SwComponentID {
+    AttestationFormat type = 1;
+    google.protobuf.Struct parts = 2;
+}
+
+message SwComponent {
+    SwComponentID id = 1;
+    google.protobuf.Struct attributes = 2;
+}
+
+message AddSwComponentsRequest {
+    repeated SwComponent info = 1;
+}
+
+message AddSwComponentsResponse {
+    Status status = 1;
+}
+
+message GetSwComponentRequest {
+    SwComponentID id = 1;
+}
+
+message GetSwComponentResponse {
+    Status status = 1;
+    google.protobuf.Struct attributes = 2;
+}
+
+message TrustAnchID {
```

Comment:
This naming seems inconsistent with `TaType` above and `TrustAnchorValue` below.  I suggest aligning the three on one of

- `TAType`, `TAID`, `TAValue`
- `TrustAnchType`, `TrustAnchID`, `TrustAnchValue`
- `TrustAnchorType`, `TrustAnchorID`, `TrustAnchorValue`

Reply:
Why do we need two `TrustAnchor`s? shouldn't the other one be removed?

Reply:
> Why do we need two `TrustAnchor`s? shouldn't the other one be removed?

yep!  I think Yogesh wasn't sure and wanted to discuss the need of the other with you before dropping it.  If you are OK with `rm trustanchor.go` let's do it :-)

Reply:
> @setrofim @thomas-fossati , Yes, I am aware that we need to remove it. It will be done in Phase 2. I just want to avoid a bulk PR and would like to go small steps at a time!

I think if we all agree that bit is not needed we might as well drop it in this same PR.

Reply:
> If you are OK with rm trustanchor.go let's do it :-)
SGTM

Reply:
ok, I am changing it now, as we speak!